agent/puppetupdate.rb

```diff
@@ -205,9 +216,15 @@ def resolve(git, env, limit=nil)
                 changes << msg
               end
             else
-              msg = reset_ref(ref, sha)
-              Log.info msg
-              changes << msg
+              time = run(git_cmd('show --quiet --format=format:%%at %s', ref)).to_i
+
+              if time < expiration_time
+                Log.info "ignoring #{dir}: older than #{expire_after_days} days"
```

Comment:
`expiration_time` is timestamp for `Time.now - 30.days`. if `time < expiration_time` then it's more than 30 days in the past and therefore "older". am i missing something?